onyo/commands/history.py

```diff
@@ -4,6 +4,7 @@
 import sys
 from pathlib import Path
 
+from shlex import quote
```

Comment:
nit: `shlex` is in the standard library, and thus should be up one line (underneath `pathlib`). The newline belongs between stdlib and other modules.